.github/workflows/kube-linter.yaml

```diff
@@ -0,0 +1,59 @@
+name: Check Kubernetes YAMLs with kube-linter
+
+on:
+  # Note that both `push` and `pull_request` triggers should be present for GitHub to consistently present kube-linter
+  # SARIF reports.
```

Comment:
@srivickynesh can you drop this workflow and use this one instead https://github.com/redhat-appstudio/tenants-config/blob/main/.github/workflows/test.yaml ?

Reply:
Done